### DISCUSSION TITLE
How to enable autolinking

### DISCUSSION THREAD

**Discussion by xobotyi** [remarkjs/remark#661] [Q&A] [answered]
I dont understand how to enable autolinking, neither can find plugin that autolink text.
 
I have such config: 
```typescript
import remark from 'remark';
import remarkBreaks from 'remark-breaks';
import remark2rehype from 'remark-rehype';
import rehypeStringify from 'rehype-stringify';
import rehypeSanitize from 'rehype-sanitize';
import a11yEmoji from '@fec/remark-a11y-emoji';

const textToHtmlParser = remark()
  .use({ settings: { resourceLink: true } })
  .use(remarkBreaks)
  .use(a11yEmoji)
  .use(remark2rehype)
  .use(rehypeSanitize)
  .use(rehypeStringify);
```

And want `https://google.com` to become `<p><a href="https://google.com">https://google.com</a></p>`

**Comment by wooorm** [comment, marked as answer]
remark-gfm

**Reply by xobotyi** [reply]
But i don't need any other functionality from gfm (table, strikethrough, tasklist) =(

**Reply by wooorm** [reply]
Why not?

**Reply by xobotyi** [reply]
Cause it is not needed =)
I need a few of markdown functionality, mostly inline and codeblocks, like discord chat messages. It seems that for now ill go with gfm but strip everything unncessary with `rehypeSanitize` schema

**Reply by wooorm** [reply]
Yes. I think that’s a better solution. Here’s a schema I’ve used before to allow only “inline” things (e.g., a paragraph with a couple of inline markup things): https://github.com/unifiedjs/unifiedjs.github.io/blob/main/generate/pipeline/description-schema.js
